recipes/mappyfile/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mappyfile" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1cc7a092e9c64cbc902a3e3d916b368cd8d72aea3f29db422036fd14fa4ede7a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - mappyfile=mappyfile.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - click   
+    - jsonref ==0.2
+    - jsonschema >=2.0   
+    - lark-parser >=0.11.3
```

Comment:
Upstream in the `setup.py` the requirement is: `lark>=1.1.5`. Please check to reconcile possible confusion between `lark` and `lark-parser`.

Reply:
@synapticarbors thanks for reviewing this. I've clarified the requirements in the comments.